framework/CodeInterfaces/PHISICS/phisicsdata.py

```diff
@@ -0,0 +1,961 @@
+          fissionMatrixInfo = self.summedDictValues(fissionMatrixInfoMPI,'FissionMatrix',self.matrixNumbering)
+        else:
+          fissionMatrixInfo = {}
+        if self.printSpatialFlux is True:
+          if self.phisicsRelap is False:
+            fluxLabelList, fluxList, matFluxLabelList, matFluxList = self.getFluxInfo(instantOutput[0])
+          if self.phisicsRelap is True:
+            fluxLabelList, fluxList = self.getFluxInfoPhiRel(instantOutput[0])
+            powerDensLabelList, powerDensList = self.getFluxInfoPhiRel(instantOutput[1])
+        if self.printSpatialFlux is False:
+          fluxLabelList = ['fluxLabel']
+          fluxList = [0.0000E+00]
+          matFluxLabelList = ['matFluxLabelList']
+          matFluxList = [0.0000E+00]
+          powerDensLabelList = ['powerDensLabelList']
+          powerDensList = [0.0000E+00]
+        depLabelList, depList, timeStepList, matList = self.getDepInfo(timeStepIndex, mrtauTimeSteps, phisicsDataDict['numberOfMPI'])
+        decayLabelList, decayList = self.getDecayHeat(timeStepIndex,mrtauTimeSteps)
+        instantDict['keff'] = keff
+        instantDict['errorKeff'] = errorKeff
+        instantDict['reactionRateInfo'] = reactionRateInfo
+        instantDict['fissionMatrixInfo'] = fissionMatrixInfo
+        instantDict['workingDir'] = self.workingDir
+        instantDict['fluxLabelList'] = fluxLabelList
+        instantDict['fluxList'] = fluxList
+        instantDict['depLabelList'] = depLabelList
+        instantDict['depList'] = depList
+        instantDict['timeStepList'] = timeStepList
+        instantDict['decayLabelList'] = decayLabelList
+        instantDict['decayList'] = decayList
+        instantDict['timeStepIndex'] = timeStepIndex
+        instantDict['mrtauTimeSteps'] = mrtauTimeSteps
+        instantDict['xsLabelList'] = xsLabelList
+        instantDict['xsList'] = xsList
+        instantDict['cpuTime'] = cpuTime
+        if self.phisicsRelap is True:
+          instantDict['powerDensList'] = powerDensList
+          instantDict['powerDensLabelList'] = powerDensLabelList
+        if self.phisicsRelap is False:
+          instantDict['matFluxLabelList'] = matFluxLabelList
+          instantDict['matFluxList'] = matFluxList
+        self.writeCSV(instantDict,timeStepIndex,mrtauTimeSteps,phisicsDataDict['jobTitle'])
+
+      if phisicsDataDict['mrtauStandAlone'] == True:
+        decayHeatMrtau = self.getDecayHeatMrtau(timeStepIndex, mrtauTimeSteps)
+        depList = self.getDepInfoMrtau(timeStepIndex, mrtauTimeSteps)
+        mrtauDict['workingDir'] = self.workingDir
+        mrtauDict['decayHeatMrtau'] = decayHeatMrtau
+        mrtauDict['depList'] = depList
+        mrtauDict['timeStepIndex'] = timeStepIndex
+        mrtauDict['mrtauTimeSteps'] = mrtauTimeSteps
+        self.writeMrtauCSV(mrtauDict)
+  
+  def summedListValues(self,list):
+    """
+      Sums up values from a given list.
+      @ In, list, list of intergers or floats
```

Comment:
if they are a list of integers or floats, this method is not useful since it can be converted in a line
```sum(list)```
if they are strings instead, the method can be converted into
``` sum ([float(elm) for elm in list])```

In addition ```list``` is a protected Python class ..change name if what I wrote above does not apply.